app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
```

Comment:
In this case, it is better to refer to the class attribute through the class name, and not like this 
`self.__class__.people[name] = self`

everything else is good)